### PR TITLE
Add browser and operating system stats to all event properties

### PR DESCRIPTION
**Pull request by i386**
# Description

Adds browser and client OS data to each analytics event

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -131,6 +131,16 @@
       <id>repo.jenkins-ci.org</id>
       <url>http://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository>
+      <id>twitter</id>
+      <url>http://maven.twttr.com</url>
```
Comment (michaelneale) [inline pom.xml]: hrm - not sure about this if it isn't in one of the other repos - not sure we should be adding another one in - doesn't seem right.

Comment (i386) [inline pom.xml]: FWIW, I have used this library for the same purpose on a previous project.

Comment (michaelneale) [inline pom.xml]: the library isn't a problem, but we have jenkins and central repos only  - and a mirror etc, this ads another dep - it really shoudl be in central

Comment (i386) [inline pom.xml]: No idea how to get it into central...

Comment (i386) [inline pom.xml]: @michaelneale I think the right thing to do here is make sure this repository is mirrored by the Jenkins project's Artifactory and then we can remove this `<repository>` entry.

I have raised [INFRA-1259](https://issues.jenkins-ci.org/browse/INFRA-1259)

Comment (i386) [inline pom.xml]: Resolved and removed.

Comment (michaelneale) [inline pom.xml]: ah ok - in that case - I think this is ok as long as it builds. I think Tyler is ok with it - so probably kosher? I am being a little conservative here ;)

**Comment by jtnord** [comment]
What is this? there is no JIRA and this appears to bring in new libraries that clash with exisiting plugins causing funky runtime dependencies?  (snakeyaml)

//cc @alvarolobato

**Pull request merged**